test/parallel/test-cluster-prof-and-logfile.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+const common = require('../common');
+const tmpdir = require('../common/tmpdir');
+
+const fs = require('fs');
+const assert = require('assert');
+const cluster = require('cluster');
+
+const TEST_TIMEOUT = 1000;
+
+const timeout = ms => new Promise(resolve => setTimeout(resolve, ms));
```

Comment:
Can you write this test without the timeout? Tests with timers are much more likely to be flakey.

Reply:
But how could i test a case when the `exit` event won't be called?

Reply:
You don't, you write the test to expect an exit event will happen. If it doesn't, the test will not complete, and eventually the test system will kill the test and fail it.